docs/troubleshooting.md

```diff
@@ -0,0 +1,199 @@
+# Troubleshooting
+
+## Introduction
+
+The purpose of this document is to provide troubleshooting steps for different
+scenarios. Where possible sections and sub-section should be created to
+classify different types of troubles.
+
+## Errors related to Service Catalog communicating with the Broker
+
+### Certificate Authority
+
```

Comment:
Let's add a little more intro to the sections. Something like:

Sometimes the service-catalog is unable to communicate with the broker because of an unknown certificate authority.

Looking at the output below, we see the broker is running but there is a `certificate signed by unknown authority` preventing the service-catalog from connecting.